tools/big_query/big_query_utils.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python2.7
```

Comment:
nit: not sure we want an dedicated directory for something as specific as big_query utils.
tools/stresstest/.... maybe?